sdk/src/main/java/io/dapr/actors/runtime/AbstractActor.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+/**
+ * TODO
+ */
+public abstract class AbstractActor {

```

Comment:
Is this the Actor base class? Name doesn't seem obvious to me.

Reply:
I renamed it to the Java way, looking at the SpringBoot code as example.

Reply:
This is mainly because IActor becomes Actor interface in Java, so name conflicts.